src/ROOTLegacyReader.cc

```diff
@@ -3,20 +3,31 @@
 #include "podio/ROOTFrameData.h"
 #include "rootUtils.h"
 
+#include <TBranch.h>
```

Comment:
Can we "ungroup" this from the stdlib ones?

Reply:
Fixed